static/src/js/components/OrderStatus/__tests__/OrderStatusItem.test.js

```diff
@@ -1942,6 +1942,129 @@ describe('OrderStatusItem', () => {
+                  },
+                  {
+                    href: 'https://harmony.uat.earthdata.nasa.gov/service-results/harmony-uat-staging/public/harmony/gdal/a75ebeba-978e-4e68-9131-e36710fb800e/006_04_00feff_asia_west_regridded.png',
+                    title: 'G1233800390-EEDTEST',
+                    type: 'image/png',
+                    rel: 'data',
+                    bbox: [
+                      -180,
+                      64.2,
+                      -169.6,
+                      71.6
+                    ],
+                    temporal: {
+                      start: '2020-01-06T08:00:00.000Z',
+                      end: '2020-01-06T09:59:59.000Z'
+                    }
+                  }
+                ],
+                status: 'complete_with_errors',
+                message: 'The job has completed with errors. See the errors field for more details',
+                request: 'https://harmony.uat.earthdata.nasa.gov/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&granuleIds=G1233800432-EEDTEST%2CG1233800431-EEDTEST%2CG1233800430-EEDTEST%2CG1233800429-EEDTEST%2CG1233800428-EEDTEST%2CG1233800427-EEDTEST%2CG1233800426-EEDTEST%2CG1233800513-EEDTEST%2CG1233800512-EEDTEST%2CG1233800425-EEDTEST%2CG1233800424-EEDTEST%2CG1233800423-EEDTEST%2CG1233800422-EEDTEST%2CG1233800421-EEDTEST%2CG1233800420-EEDTEST%2CG1233800419-EEDTEST%2CG1233800418-EEDTEST%2CG1233800417-EEDTEST%2CG1233800511-EEDTEST%2CG1233800510-EEDTEST%2CG1233800416-EEDTEST%2CG1233800415-EEDTEST%2CG1233800414-EEDTEST%2CG1233800413-EEDTEST%2CG1233800412-EEDTEST%2CG1233800411-EEDTEST%2CG1233800410-EEDTEST%2CG1233800409-EEDTEST%2CG1233800408-EEDTEST%2CG1233800509-EEDTEST%2CG1233800508-EEDTEST%2CG1233800407-EEDTEST%2CG1233800406-EEDTEST%2CG1233800405-EEDTEST%2CG1233800404-EEDTEST%2CG1233800403-EEDTEST%2CG1233800402-EEDTEST%2CG1233800401-EEDTEST%2CG1233800400-EEDTEST%2CG1233800399-EEDTEST%2CG1233800507-EEDTEST%2CG1233800506-EEDTEST%2CG1233800398-EEDTEST%2CG1233800397-EEDTEST%2CG1233800396-EEDTEST%2CG1233800395-EEDTEST%2CG1233800394-EEDTEST%2CG1233800393-EEDTEST%2CG1233800392-EEDTEST%2CG1233800391-EEDTEST%2CG1233800390-EEDTEST&subset=time(%222020-01-06T08%3A18%3A35.096Z%22%3A%222020-01-10T20%3A38%3A58.262Z%22)&format=image%2Fpng&outputCrs=EPSG%3A4326',
+                progress: 100,
+                username: 'rabbott',
+                createdAt: '2020-09-10T13:50:22.372Z',
+                updatedAt: '2020-09-10T13:50:22.372Z'
+              }
+            }],
+            isLoaded: true
+          }
+        })
+
+        expect(enzymeWrapper.hasClass('order-status-item--complete')).toEqual(true)
+        expect(enzymeWrapper.hasClass('order-status-item--in_progress')).toEqual(false)
+        expect(enzymeWrapper.hasClass('order-status-item--failed')).toEqual(false)
+
+        const header = enzymeWrapper.find('.order-status-item__header')
+        expect(header.find(ProgressRing).props().progress).toEqual(100)
```

Comment:
Fixed looks like its messed up all over this test. Since we'll need to rewrite this in RTL at some point in the medium term I'm not gonna spend time doing that now